bika/lims/browser/js/bika.lims.analysisservice.js

```diff
@@ -604,10 +671,15 @@ function AnalysisServiceEditView() {
                 if (data.objects.length > 0) {
                     $(calculation_select_element).append('<option value="'+data.objects[0].UID+'">'+data.objects[0].Title+'</option>');
                     $(calculation_select_element).val(data.objects[0].UID);
+                    $(calculation_select_element).find('option').prop('selected',true);
+                    $(calculation_select_element).find('option').attr('selected','selected');
```

Comment:
$(calculation_select_element).find('option').prop('selected',true);
$(calculation_select_element).find('option').attr('selected','selected');

Do you need both?